test/schemas/10-10EZ/schema.spec.js

```diff
@@ -14,7 +16,7 @@ function definitionValidator(field) {
     $schema: 'http://json-schema.org/draft-04/schema#',
     type: 'object',
     properties: {
-      field: applicationSchema.definitions[field],
+      field: definitions[field],
```

Comment:
definitionValidator should test definitions in the schema, if you want to test definitions from definitions.js put them in definitions.spec.js. if the definition is already tested in definitions.spec.js then i would just delete the spec from this file

Reply:
Okay. I removed the duplicated schema spec tests and beefed up the `definitions.spec.js` tests.